content/03-ventures/03-ideation-breakfast.md

```diff
@@ -0,0 +1,35 @@
+# Ideation Breakfast
+
+The purpose of the ideation breakfast is to discuss and decide on the execution of internal product ideas. Ideas can be submitted in the form of pitches to be evaluated at the ideation breakfast. After the ideation breakfast, the chosen product idea can be scheduled, planned and executed.
+
+## Formulated document
+
+Ideas are not presented in a spoken form but in a written pitch. The pitch needs to be written out in continuous text and should be self-explanatory without having further information. The structure can begin with a summary of the idea, followed by a description of the problem that led to the idea and then the solution. The deadline for submitting pitches is until the eve of each Ideation Breakfast. The deadline for submitting pitches is the evening before the ideation breakfast.
+
+[Example Pitch](https://www.notion.so/0727184bf5b840afa2fa4ea0c85ee864)
+
+## Read session
+
+Before we go into the discussion of ideas, we take the time to read all pitches from front to back. This helps to make sure, that everyone is on the same page, understands the product idea in detail and had time to think about it. Everyone can take notes and questions that can be addressed later.
+
+## Questions
+
+After all have read through the pitches, the author of the pitch has to answer open questions. The answers help to deepen the understanding of the product idea and diminish uncertainty.
+
+## Classify Impact / Effort
+
+In order to decide which product idea will make it into the next cycle, it is best to arrange the ideas according to impact and effort. The impact could include both economic potential and the impact of the problem.
+
+What the process looks like this:
+
+1. Draw an Impact / Effort Scale
+2. Write the name of one product idea on a post-it
+3. Move your hand over the middle of the impact scale and ask, "Is the impact higher or lower?"
+4. The moderator should then move the note up or down until the others stop saying higher or lower
+5. Same procedure with the Effort scale
+6. Repeat the process for all other ideas
+7. The moderator should mark all ideas in the top left corner with a contrast dot
+
+## Prioritizing ideas
+
+If all ideas are hanging on the wall and have been discussed can be voted with sticky dots. The ideas are then sorted on the wall according to most votes.
```

Comment:
```suggestion
When all ideas are hanging on the wall and have been discussed, we vote for them with sticky dots and sort on the wall by the number of votes.
```

Reply:
Great, much better wording. I just added a "them" after "and sort...". I felt it was missing there. 👍